src/KubeOps/Operator/Services/IResourceTypeService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace KubeOps.Operator.Services
+{
+    public interface IResourceTypeService
```

Comment:
Since the interface is public, it should contain documentation (all public elements)

Reply:
I made this internal since there is no point where the service or its interface ever get accessed or returned by anything but this library.

If that is the rule (I wasn't sure if you were strictly following that), shouldn't `IResourceServices<TEntity>` be documented? I didn't take the time to write out documentation because I was following the "rules" that seemed to be obeyed the other service interface in that namespace.